packages/cli/src/commands/test.js

```diff
@@ -26,10 +26,13 @@ export const builder = (yargs) => {
 
 export const handler = async ({ side }) => {
   const { base: BASE_DIR } = getPaths()
+
+  const DB_URL = process.env.TEST_DATABASE_URL || 'file:/test.db'
```

Comment:
I think the "failover" path might be incorrect here.

Reply:
But, I'm wondering where we should place it, maybe `./api/prisma/test.db`?

Reply:
Ideally it's in should be in a temporary location, but we don't really have one of those yet.